bin/compile-extension-mongo

```diff
@@ -8,7 +8,7 @@ travis_time_start
 if [[ ! $VERSION =~ ^7 && ! $VERSION =~ ^master$ ]]; then
 	git clone https://github.com/mongodb/mongo-php-driver-legacy.git
 	pushd mongo-php-driver-legacy
-	git checkout 1.6.12
+	git checkout 1.6.16
```

Comment:
Is there any reason this script builds from source instead of calling `pecl_install mongo` as is done in [compile-extension-mongodb](https://github.com/travis-ci/php-src-builder/blob/default/bin/compile-extension-mongodb#L18) for PHP 5.x?

Reply:
I can't remember the exact reason, but that was the original implementation. It must have been problematic, and I have this commit to fix it. https://github.com/jmikola/php-src-builder/commit/d11dbca369e9e0bc390ca3070bc2f998554c56f5